duo_client/admin.py

```diff
@@ -2876,3 +2876,96 @@ def sync_user(self, username, directory_key):
             '/admin/v1/users/directorysync/{directory_key}/syncuser').format(
                 directory_key=directory_key)
         return self.json_api_call('POST', path, params)
+
+    def get_trust_monitor_events_generator(
```

Comment:
sorry for the nit pick, but could you rename this to `get_trust_monitor_events_iterator`
the other client functions use iterator instead of generator in the function name and it's probably best we just stay consistent.

Reply:
Yeah that's a good call for consistency.  I went ahead and updated the method name to better match the others.